### DISCUSSION TITLE
Sim 2 Not Loading

### DISCUSSION THREAD

**Discussion by jbain1029** [SimDynamX/SpaceTeamsPRO#65] [2024 Lunar SAR Competition Q&A] [answered]
Sim 1 is still loading fine, but when I try to access sim 2, it gets stuck on the 'loading sim' screen. I have tried adding the sim json file to my original and also tried attaching the entire new repository to STP, but neither solution worked.

**Comment by connorjak** [comment]
There are several changes in many files for Task 2; the best way to update is to back up your current files (can move them to a folder), paste all of the 1.1.0 STU_Lunar_SAR files on top of your existing repo, and bring pieces back from your Task 1 python scripts to reapply back for Task 2.

**Reply by connorjak** [reply, marked as answer]
Keep in mind you will need to redo changing the `STU_Competition2` folder names to match the Repos folder they are now in.
